### ISSUE TITLE
how about include files?

### ISSUE THREAD

**Issue by diuming** [prsolucoes/pdfium-lib#9]
It is a good tool for building static library.
I have 2 problems.
1)how about include files(collect include files)?
2)how do i modify it for supporting MacOSX10.15.sdk?
3)what is difference between libpdfium.a and libpdfium_x64.a?

thanks

**Comment by prsolucoes** [comment]
Hi,

1. Check tutorial here:
https://github.com/prsolucoes/mobile-pdfium/blob/master/HOW_TO_INCLUDE_FILES.md

2. SDK 10.15 is already support. The sdk_info.py script already get the correct path. It execute command `xcrun -sdk macosx --show-sdk-path` that will return:

>/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk

3. You can check the library arch using "lipo -info" and see what is returned. But one is for mobile and other for simulator.

Thanks.

**Comment by prsolucoes** [comment]
Hi,

I think that you are asking about SDK 10.15 because has a fixed version of 10.14 checking in the code.

The python script from pdfium don't check min version but their IF is checking different version.

To solve it execute:

```
export FORCE_MAC_SDK_MIN=10.15
```

and your command again, example:

```
python make.py run build-macos
```

Thanks.

**Comment by prsolucoes** [comment]
Hi,

It was solved? Can i close this?

Thanks.
